### ISSUE TITLE
Return error on absent file

### ISSUE THREAD

**Issue by m9aertner** [editorconfig-checker/editorconfig-checker#157]
It would be great to fail fast on bad command line input. When the file given is not there, there should not be a zero return code, but 1.

    $ ec no-such-file
    Could not get the ContentType of file: no-such-file
    stat no-such-file: no such file or directory
    
    $ echo $?
    0                   <------ should ideally be 1

    $ ec --version
    2.3.4

**Comment by mstruebing** [comment]
Hey @m9aertner thinking about that issue: 
That would mean that a call like `ec existing-file non-existing-file` would also result in exiting the program. Do you think that should happen?

**Comment by m9aertner** [comment]
Yes, I believe that is "expected behaviour" with many (if not all) unixoid tools, such as `grep`:

    $ echo ABC > abc.txt
    
    $ grep -c ABC abc.txt
    1
    
    $ echo $?
    0
    
    $ grep -c ABC abc.txt no-such-file.txt
    abc.txt:1
    grep: no-such-file.txt: No such file or directory
    
    $ echo $?
    2

Here, the bad file name also causes the program's return code to indicate "error" (2, != 0), even though a good file is present.

Also note that the order of arguments in the last example is not relevant. All good files are processed, all bad files are reported, and if at least one bad file is present, the return code is 2, not 0.

    $ grep -c ABC no-such-file.txt abc.txt
    grep: no-such-file.txt: No such file or directory
    abc.txt:1

    $ echo $?
    2

**Comment by theoludwig** [comment]
I believe it was fixed in #599
